docs/src/getting-started.rst

```diff
@@ -181,41 +229,41 @@ library includes the basic classes needed to write |cdk| stacks and apps.
 
 .. _define_app:
 
-Define the |cdk| App
-====================
+Defining the |cdk| App
+======================
 
-|cdk| apps are modeled as classes which extend the :py:class:`App <@aws-cdk/cdk.App>`
-class. Let's create our first, empty **App**:
+|cdk| apps are classes that extend the :py:class:`App <@aws-cdk/cdk.App>`
+class. Create an empty **App**:
 
 .. tabs::
 
     .. group-tab:: JavaScript
 
-        In **index.js**:
+        Create the file **hello-cdk.js** in the **bin** folder:
```

Comment:
I think it's clearer if you write it as `bin/hello-cdk.js `

Reply:
sure